README.md

```diff
@@ -97,6 +97,19 @@ Overview of alternative open source front-ends for popular internet platforms (e
 
 - [ytfzf](https://github.com/pystardust/ytfzf): A POSIX script that helps you find Youtube videos (without API) and opens/downloads them using mpv/youtube-dl
 
+- [ytcc](https://github.com/woefe/ytcc): Command line tool to keep track of your favorite playlists on YouTube and many other places. Can import youtube subscriptions from Google Takeout and provide them as an RSS feed for your favorite reader.
+
+- [ytmdesktop](https://github.com/ytmdesktop/ytmdesktop): Cross-platform (Windows, Mac, and Linux) desktop app for YouTube Music. Has a (proprietary?) remote control app for android.
+
+- [AudioTube](https://invent.kde.org/plasma-mobile/audiotube): Audiotube is a client for YouTube Music. Plasma-mobile project with an interface designed for Linux phones.
+
+- [smtube](http://smplayer.sourceforge.net/smtube.php): Stand-alone YouTube video player
```

Comment:
When I open the link http://smplayer.sourceforge.net/smtube.php I get redirected to https://www.smtube.org .

The source code seems to be hosted at https://github.com/smplayer-dev/smtube .